objects/src/errors.rs

```diff
@@ -33,7 +33,10 @@ pub enum AccountError {
     NonFungibleAssetNotFound(NonFungibleAsset),
     NotAFungibleFaucetId(AccountId),
     NotANonFungibleAsset(Asset),
-    SeedDigestTooFewTrailingZeros,
+    SeedDigestTooFewTrailingZeros {
+        expected: u32,
+        got: u32,
+    },
```

Comment:
minor nit: we usually use `actual` rather than `got` (e.g., in `AccountCodeTooManyProcedures` above). Might be good to rename for the sake of consistency.

Reply:
updated